app/views/user_mailer/peer_mail.html.erb

```diff
@@ -15,7 +15,11 @@
 		<h4>Your mini-meetup for this month: </h4>
 		<ul>
       <% @peers.each do |peer| %>
-        <li><%= peer.first_name%> <%= peer.last_name%> <%= mail_to( peer.email)%> (Ask me about: <% peer.top_3_interests.join(", ") %>)</li>
+        <li>
```

Comment:
We updated how we group users, and I didn't catch the reference here on lines 11 and 32. Would you mind updating the text to reflect that we only group by location now?